azure_arc_servers_jumpstart/automanage/artifacts/arc_automanage.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Environment Variables
+export automanageAccountName='<Name of your Azure Automanage Account>'
+export location='<Azure Region where your Azure Arc enabled server is registered>'
+export resource_group='<Azure Resource Group where your Azure Arc enabled server is registered>'
+export machineName='<Name of your Azure Arc enabled Server>'
+export profile='<environment of your Azure Arc enabled server Production or Dev/Test>'
+
+# Replace environment variables in parameters file
+
+sed -i "s/<name of your Automanage Account identity>/$automanageAccountName/" automanage_account.parameters.json
+sed -i "s/<azure region>/$location/" automanage_account.parameters.json
+
+# Create Azure Automanage Account
+
+az deployment group create --resource-group $resource_group --template-file automanage_account.json --parameters automanage_account.parameters.json
+
+sleep 30
```

Comment:
I think checking the status here instead of _sleep_ is more reliable

Reply:
wasn't necesary left from testing